core/CronArchive.php

```diff
@@ -868,7 +868,7 @@ private function invalidateRecentDate($dateStr, $idSite)
             return;
         }
 
-        $this->logger->info("  Will invalidate archived reports for $dateStr in site ID = {idSite}'s timezone ({date}).", [
+        $this->logger->debug("  Will invalidate archived reports for $dateStr in site ID = {idSite}'s timezone ({date}).", [
```

Comment:
if these things are not supposed to happen often, it would be useful to log as `info`? in my ticket i just meant the 2 extra lines with no useful info displayed for each site, i didn't mean we necessarily remove every line as these look useful (as long as they aren't expected to be triggered all the time)